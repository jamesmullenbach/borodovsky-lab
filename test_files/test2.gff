NC_000913	verified	CDS	2	30	.	+	0	gene_id=test start_codon=ATG
