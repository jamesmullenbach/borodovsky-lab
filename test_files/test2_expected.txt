gene average: 1.552
gene variance: 7.420
